nautobot_chatops/choices.py

```diff
@@ -44,3 +44,19 @@ class CommandTokenPlatformChoices(ChoiceSet):
     WEBEX = "webex"
 
     CHOICES = ((MATTERMOST, "Mattermost"),)
+
+
+class PlatformChoices(ChoiceSet):
```

Comment:
Seems to me duplicit with CommandTokenPlatformChoices.

Reply:
Yeah, I realize that now, I think in my mind I was going to replace CommandTokenPlatformChoices with PlatformChoices.